spanner/integration_test.go

```diff
@@ -262,6 +271,10 @@ func initIntegrationTests() (cleanup func()) {
 	if err != nil {
 		log.Fatalf("Cannot get any instance configurations.\nPlease make sure the Cloud Spanner API is enabled for the test project.\nGet error: %v", err)
 	}
+	configName := config.Name
+	if instanceConfig != "" {
```

Comment:
If `instanceConfig` is not empty, there is no need to call `ListInstanceConfigs`. 

```go
        var configName string
	if instanceConfig != "" {
		configName = fmt.Sprintf("projects/%s/instanceConfigs/%s", testProjectID, instanceConfig)
	} else {
	        configIterator := instanceAdmin.ListInstanceConfigs(ctx, &instancepb.ListInstanceConfigsRequest{
		        Parent: fmt.Sprintf("projects/%s", testProjectID),
	        })
	        config, err := configIterator.Next()
	        if err != nil {
		        log.Fatalf("Cannot get any instance configurations.\nPlease make sure the Cloud Spanner API is enabled for the test project.\nGet error: %v", err)
	        }
	        configName := config.Name
        }
```

Reply:
Added an if-else.